### PR TITLE
add kms decryption

### PR DESCRIPTION
**Pull request by MancunianSam**
- Decrypt the environment variables using kms
- Pass the endpoint to the kms client

### CODE REVIEW

File: src/test/resources/application.conf [hunk 1]
```diff
@@ -1,6 +1,8 @@
 driver="org.postgresql.Driver"
-username="tdr"
-password="password"
-url="jdbc:postgresql://localhost:5432/consignmentapi"
+username="dGRy" # tdr
```
Comment (TomJKing) [inline src/test/resources/application.conf:2]: Is it worth adding comment to explain what the string in the comment is?

File: src/test/resources/application.conf [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+password="cGFzc3dvcmQ=" # password
+url="amRiYzpwb3N0Z3Jlc3FsOi8vbG9jYWxob3N0OjU0MzIvY29uc2lnbm1lbnRhcGk=" # jdbc:postgresql://localhost:5432/consignmentapi
 consignment-api-user = "consignment_api_user"
```
Comment (TomJKing) [inline src/test/resources/application.conf:5]: Do all the values need to be encrypted? If not maybe state this in the README

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a couple of minor comments

File: src/test/resources/application.conf (hunk 1 above)
Comment (MancunianSam) [reply src/test/resources/application.conf:2]: I've added in the same comment as the others.

File: src/test/resources/application.conf (hunk 2 above)
Comment (MancunianSam) [reply src/test/resources/application.conf:5]: Only ones that come from environment variables in main application.conf. I've put a line in the README to say this.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**